core/controllers/base.py

```diff
@@ -295,8 +299,78 @@ def dispatch(self):
                 self.handle_exception(e, self.app.debug)
                 return
 
+        handlers_has_passed_schema_validation = True
+        try:
+            self.validate_args_schema()
+        except self.InvalidInputException as e:
+            self.handle_exception(e, self.app.debug)
+            handlers_has_passed_schema_validation = False
+        except NotImplementedError as e:
+            self.handle_exception(e, self.app.debug)
+            handlers_has_passed_schema_validation = False
+        if handlers_has_passed_schema_validation is False:
```

Comment:
Add a line break above this one so that it's clearly separate from the try/except.

Reply:
Done